data/src/test/java/com/shopapp/data/impl/AuthRepositoryTest.kt

```diff
@@ -212,7 +212,6 @@ class AuthRepositoryTest {
         observerCustomer.assertError(error)
     }
 
```

Comment:
Please, add tests for `updateAccountSettings` in AuthRepository.

Reply:
done